src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```
@@ -1179,117 +1150,117 @@ protected function buildDeployment(
 
-        if (!($deployment instanceof Document)) {
+        if (! ($deployment instanceof Document)) {
             throw new Exception('deployment must be an instance of Document');
         }
     }
 
     protected function getRuntime(Document $resource, string $version): array
     {
         $runtimes = Config::getParam($version === 'v2' ? 'runtimes-v2' : 'runtimes', []);
-        $key =  $resource->getAttribute('runtime');
+        $key = $resource->getAttribute('runtime');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)